runhouse/servers/cluster_servlet.py

```diff
@@ -376,8 +379,18 @@ async def aperiodic_autostop_check(self):
                     logger.debug("Successfully updated autostop")
 
             except Exception as e:
+
                 logger.error(f"Autostop check has failed: {e}")
 
+                # killing the cluster servlet only if log level is debug: this indicates that the cluster is created
+                # during test, or by a runhouse team member
+                log_level = os.getenv("RH_LOG_LEVEL")
+                if log_level.lower() == "debug":
```

Comment:
I don't think so, because I added a fix for syncing the `RH_LOG_LEVEL` to the cluster: if there is no `RH_LOG_LEVEL` local env var, we sync the default log level (currently synced to 'INFO'). I tested this logic, and it seems to be working. Nevertheless, I think that we could use here the `DEFAULT_LOG_LEVEL` as well, just to double check that log_level.lower() does not causing a breakage. 

wdyt?